report.md

```diff
@@ -111,5 +152,24 @@ Notice that this data has 2-anonymity with respect to the attributes `Age`, `Gen
 
 ## Subsystem 4 (Secure Transfer)
 
+### Overview
+
+The other team will be given a special account (which is a type of user, such as `therapist` or `patient`) that can only perform one action: Upload their database to our database. 
+
+### Interface
+
+Upon logging into the special account, the other team will be redirected to a page whereby they can upload their database, image and video files separately. Our current database design is such that the image and video files are stored in a separate file system. The database stores the image and video file names. When a record is retrieved from the database, the server will also retrieve the corresponding image and video files based on their names.
+
+The data can be in tab-separated value (tsv), with the following values:

```

Comment:
Is this a thing?? I though it was still CSV just the delimeter was tab hahaha. Can I clarify what is subtype?

Reply:
Hmm Im not sure this part came with the branch when I branch from master last night. This is under Subsytem 4 and seem to have been merged into the master already. But I do feel that tsv would be easier to handle here.
From the example given by prof,  type would be the main category(eg. Readings) and subtype would be Blood Pressure(BP) readings